system/security/LdapSecurity/ldapsecurity.cpp

```diff
@@ -1556,6 +1556,7 @@ bool CLdapSecManager::clearPermissionsCache(ISecUser& user, IEspSecureContext* s
             return false;
         }
         m_permissionsCache->flush();
+        PROGLOG("Permissions cache cleared by admin user %s", user.getName());
```

Comment:
minor: not really related to the purpose of this PR I assume?

Reply:
In the Jira,Fern  mentioned that Clear Permissions Cache didn't log anything, so I added this. If you prefer I could create a separate JIra/PR